Package.swift

```diff
@@ -19,11 +22,24 @@ import PackageDescription
 
 let package = Package(
     name: "SwiftKuerySQLite",
-    targets: [Target(name: "SwiftKuerySQLite", dependencies: [.Target(name: "CSQLite")]),
-              Target(name: "CSQLite")
+    products: [
+        .library(
+            name: "SwiftKuerySQLite",
+            targets: ["SwiftKuerySQLite"]
+        )
     ],
     dependencies: [
-        .Package(url: "https://github.com/IBM-Swift/Swift-Kuery.git", majorVersion: 0, minor: 13),
-        ],
-    exclude: ["Configuration", "Scripts"]
+        .package(url: "https://github.com/IBM-Swift/Swift-Kuery.git", from: "1.0.0"),
+        .package(url: "https://github.com/NocturnalSolutions/CSQLite.git", from: "0.2.0")
```

Comment:
Should we import `CSQLite` into  `Swift-Kuery-SQLite` or create a new repo for `CSQLite`?

Reply:
Having a separate repo for the bridge code seems to be a standard practice, at least in regards to Kuery. CSQLite itself is more or less just a copy of IBM's [CMySQL](https://github.com/IBM-Swift/CMySQL) with the paths changed to point to the SQLite headers instead of the MySQL ones. I'm not sure if there is a technical reason for this or if it's just done to help out someone else that might want to bridge to MySQL/SQLite in their project but not necessarily use Kuery proper.

Reply:
We have to decided to keep it inside `Swift-Kuery-PostgreSQL`, could you remove the dependency and declare a target : 
```
.target(
  name: "CSQLite"
)
```

Reply:
Ya know, as I went to do this, I eventually found out that the code for a CSQLite target was already in the project. At commit 11187ea, when I changed Package.swift to Swift 4, I removed that target and I suppose at that point created the separate CSQLite target.

I can't remember why I did this. I suppose that I just had some sort of problem, and figured that since Swift Kuery MySQL had the separate repo, this one had to have the separate repo too.

At any rate, I fixed this, but won't push a fix right now because I'm getting a test failure when I run tests. It's too late for me to look into it tonight. I'll try again tomorrow.